docs/dev/Lua API.rst

```diff
@@ -4352,6 +4352,15 @@ There are the following predefined frame style tables:
 
   A frame suitable for overlay widget panels.
 
+* ``MEDIUM_FRAME_2``
+
+  This version of MEDIUM_FRAME does *not* have a visible ``DFHack`` signature on it,
+  so it must not be used as the most external frame for a DFHack-owned UI.
+
+* ``BOLD_FRAME``
+
+  A frame suitable for overlay widget panels.
```

Comment:
`BOLD_FRAME` may have a good place in the frame lineup, but I've specifically been using `MEDIUM_FRAME` for overlay panels. Overlay panels are often in crowded spaces where thick borders make the layout too crowded. The buildingplan overlay is special since it's on its own without vanilla elements around it, but even there why not `PANEL_FRAME`?

Reply:
honestly-- i just copied `MEDIUM_FRAME`'s description because i didn't know what to write. :flushed:

what i meant to write was something along the lines of: 
"use this when you need your panel to have a bigger impact on screen", or
"a frame suitable for _bold_ panels" in the sense that it's an important panel/one you should notice.

i didn't know how to word that and i thought it'd be best to discuss it.

as for why not `PANEL_FRAME`... i think that in planneroverlay's case it complements vanilla's UI the best.
![Capture1](https://user-images.githubusercontent.com/5853667/229289115-70f1fbf0-7d82-429f-bdbb-93124877e4e1.PNG) ![Capture2](https://user-images.githubusercontent.com/5853667/229289119-0d92db02-7c26-4c05-b70f-31afec77f1ef.PNG)

`PANEL_FRAME`'s square angles and its slightly lighter shade of yellow feel a bit out of place, imo.

i feel that, generally, a `PANEL_FRAME` border should encompass the whole contents of a window. :thinking: 
Since part of this window is already using a `MEDIUM_FRAME` external border, having a panel using a `PANEL_FRAME` border tacked under it feels a bit jarring. 
In this case `BOLD_FRAME` looks more, like, "organic", less "tacked-on" somehow. _And_ in this case also helps visually separating its contents from the controls you can directly interact with.

I agree that `BOLD_FRAME` could be considered redundant in other places... But maybe we can find another use for it for windows like `confirm`'s confirmation dialogs. 🤔

Reply:
you're talking above my pay grade here. I trust you on the visual style decisions.